certified-connectors/AssemblyAI/apiDefinition.swagger.json

```diff
@@ -1065,8 +1121,124 @@
           "description": "The language of your audio file. Possible values are found in Supported Languages.\nThe default value is 'en_us'.",
           "x-ms-visibility": "important",
           "type": "string",
+          "enum": [
+            "en",
+            "en_au",
+            "en_uk",
+            "en_us",
+            "es",
+            "fr",
+            "de",
+            "it",
+            "pt",
+            "nl",
+            "af",
+            "sq",
+            "am",
+            "ar",
+            "hy",
+            "as",
+            "az",
+            "ba",
+            "eu",
+            "be",
+            "bn",
+            "bs",
+            "br",
+            "bg",
+            "my",
+            "ca",
+            "zh",
+            "hr",
+            "cs",
+            "da",
+            "et",
+            "fo",
+            "fi",
+            "gl",
+            "ka",
+            "el",
+            "gu",
+            "ht",
+            "ha",
+            "haw",
+            "he",
+            "hi",
+            "hu",
+            "is",
+            "id",
+            "ja",
+            "jw",
+            "kn",
+            "kk",
+            "km",
+            "ko",
+            "lo",
+            "la",
+            "lv",
+            "ln",
+            "lt",
+            "lb",
+            "mk",
+            "mg",
+            "ms",
+            "ml",
+            "mt",
+            "mi",
+            "mr",
+            "mn",
+            "ne",
+            "no",
+            "nn",
+            "oc",
+            "pa",
+            "ps",
+            "fa",
+            "pl",
+            "ro",
+            "ru",
+            "sa",
+            "sr",
+            "sn",
+            "sd",
+            "si",
+            "sk",
+            "sl",
+            "so",
+            "su",
+            "sw",
+            "sv",
+            "tl",
+            "tg",
+            "ta",
+            "tt",
+            "te",
+            "th",
+            "bo",
+            "tr",
+            "tk",
+            "uk",
+            "ur",
+            "uz",
+            "vi",
+            "cy",
+            "yi",
+            "yo"
+          ],
           "x-ms-summary": "Language Code"
         },
+        "language_detection": {
+          "description": "Enable Automatic language detection, either true or false.",
+          "type": "boolean",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Language Detection"
+        },
+        "speech_model": {
```

Comment:
Kindly confirm that the existing flows will not be broken from the below swagger validation error. 

An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/CreateTranscript/parameter/body/schema/speech_model/enum

Reply:
The enum value that was removed was already removed from the API, so it would be broken if someone used it.
However, this connector hasn't been released yet, so no users will be broken.